services/src/main/java/org/keycloak/protocol/oidc/endpoints/UserInfoEndpoint.java

```diff
@@ -136,7 +137,9 @@ public Response issueUserInfoPost() {
 
         // Fallback to form parameter
         if (accessToken == null) {
-            accessToken = request.getDecodedFormParameters().getFirst("access_token");
+            MultivaluedMap<String, String> formParams = request.getDecodedFormParameters();
+            checkParameterDuplicated(formParams);
+            accessToken = formParams.getFirst("access_token");
```

Comment:
The existing UserInfoEndpoint only checks "access_token" and igores other unknown parameters.

This PR checks duplication of "access_token" and other unknows parameters.

Another option is to check only duplication of "access_token" and ignores unknow parameters.

@m-takai 
Which option do you prefer?

If checking duplication of unknown parameters, it should be checked regardless of "acess_token" being null or not.

Reply:
Thank you for you review and a valuable comment.
Although OIDC specification does not specify how treat unknown parameters in UserInfo endpoint, the other endpoints such as authorization endpoint ignore unknown parameters followed by RFC6749. Thus, I fix the PR code to check only duplication of "access_token".